modules/sdk/src/utils/connext.ts

```diff
@@ -686,3 +691,20 @@ export const getChannelForChain = async (
   }
   return channel as FullChannelState;
 };
+
+export const sendTransaction = async (
```

Comment:
Let's call this `onchainTransfer` or something, since `sendTransaction` means something different.

Reply:
make sense!